octoprint_mrbeam/iobeam/lid_handler.py

```diff
@@ -840,71 +841,47 @@ def updt(val_prev, val_new, func=np.average, **kw):
 		except Exception as ex:
 			self._logger.exception('Exception_in-_save__s_for_analytics-_{}'.format(ex))
 
-	def send_last_img_to_analytics(self):
-		self._flag_send_img_to_analytics = True
-
-	def _send_last_img_to_analytics(self, img, trigger, markers, missed, min_pix_amount, analytics, force_upload=False, notify_user=False):
-		self._flag_send_img_to_analytics = False
-		t = threading.Thread(target=self._send_last_img_to_analytics_threaded,
-							 name='send_last_img_to_analytics',
-							 kwargs={'img': img,
-									 'trigger': trigger,
-									 'markers': markers,
-									 'missed': missed,
-									 'min_pix_amount': min_pix_amount,
-									 'analytics_data': analytics,
-									 'force_upload': force_upload,
-									 'notify_user': notify_user,
-									 })
-		t.daemon = True
-		t.start()
-
-	def _send_last_img_to_analytics_threaded(self, img, trigger, markers, missed, min_pix_amount, analytics_data, force_upload=False, notify_user=False):
-		try:
-			if img is not None:
-				img_format = 'jpg'
-				_, img = cv2.imencode('.{}'.format(img_format), img, [int(cv2.IMWRITE_JPEG_QUALITY), 80])
-				img = base64.b64encode(img)
+	@get_thread(name='send_last_img_to_analytics',)
+	def send_last_img_to_analytics(self, force_upload=False, trigger='user', notify_user=False):
+		raw_path = os.path.join(os.path.dirname(self.final_image_path), 'debug/raw.jpg')
+		latest = cv2.imread(raw_path)
```

Comment:
Image for the analytics is read from the `debug/raw.jpg` file, which corresponds to the last image. It should match the latest `self.analytics` as well